packages/perseus/src/widgets/radio/choice.tsx

```diff
@@ -346,6 +346,7 @@ const styles = StyleSheet.create({
         padding: intermediateCheckboxPadding,
         paddingTop: 0,
         marginLeft: 54,
+        color: Color.offBlack64,
```

Comment:
There are two references to rationales in our `.less` styles: 

1. https://github.com/Khan/perseus/blob/cfff6ad16a3923418a3c75816af7c99689b94da4/packages/perseus/src/styles/articles.less#L237
2. https://github.com/Khan/perseus/blob/cfff6ad16a3923418a3c75816af7c99689b94da4/packages/perseus/src/styles/articles.less#L386

These are in Article styles. Just wanted to check if you verified this in Articles. The styles you've added here likely override those .less styles, but it might be useful to clean them up just so we don't introduce a regression at some point in the future if we move this color style for some reason.